### ISSUE TITLE
Plan B issues

### ISSUE THREAD

**Issue by kyraith** [WarEmu/WarBugs#18121]
**Expected behavior and actual behavior:**
Plan B no longer reliably rotates through stances during Forts and Cities, often getting stuck in the 3rd stance (when you are not) or reseting to the 1st stance (when you should be in 3rd)

Issue seemed to start with the hotfix that patched the Chosen Crit Reduction ability

Some more info here:
https://www.returnofreckoning.com/forum/viewtopic.php?f=66&t=33007&p=449415#p449415

**Steps to reproduce the problem:**
- Use Plan B on a Black Orc or Swordmaster in Fort or City (high lag areas?)
- Note the unreliability of the addon
- Throw keyboard

Sorry I realize this is for an Addon and not the client itself, but it is an Addon many depend on and I dont think anyone supports the Addon anymore; and I think a Client/Server change broke it

**Comment by kyraith** [comment]
It is incredibly difficult to give my party Whispering Winds when my stances keep reseting before I can get to the third stance

**Comment by lifeson889** [comment]
Yes this bug has been around for a couple of months now. Doesn't just happen just in laggy fights, I have had it in city too. CC chains seem to confuse the stance mechanic too. 
Skills get grayed out out of sequence, sometimes they are actually available because the stance is valid and sometimes not. Makes playing SM pretty miserable at the moment.

**Comment by L0hki** [comment]
This issue is still happening. I recently started playing BO and SM and I noticed that during large fights, especially when blasting pots are in heavy use the stance got de-synchronized. It means that in career mechanic perfect balance / best plan is displayed when in fact basic stance is active. This is especially visible for Plan B addon users such us myself.

**Comment by PrincessCake** [comment]
Still happening and ALOT, like every 30sec or so.
If using PlanB it greys out your bar and you can't do anything to get "unstuck" except using suddenshift to reset your bar.
Tries to wait 5 minutes to see if it would reset by itself, it didn't.

![planB_problem](https://user-images.githubusercontent.com/85670427/121484071-2b7da580-c9cf-11eb-9854-cce9e1d90895.png)

**Comment by baroke** [comment]
Adding some feedback here, this is on my SM, in improved balance stance as per the Core stance widget (on top of my character) and i get "Requires improved balance" when i try to use an ability. You can also see the toolbar below (plan b) with buttons disabled, but i agree that the problem is not on Plan B.

![image](https://user-images.githubusercontent.com/20115114/126082576-bf43cb56-8b75-4142-8a52-7f05b3d6919b.png)

**Comment by PrincessCake** [comment]
Problem still exist
![planB sux](https://user-images.githubusercontent.com/85670427/175805803-d466dda3-3a7f-4b38-ba61-9f0f17cdbdd8.png)
s

**Comment by MarkusHeb** [comment]
The problem still persists. Please look into this if possible, 2 classes are currently very clunky to play.

**Comment by Koenigswurm** [comment]
Please help us, you are our only hope
https://www.youtube.com/watch?v=5cc_h5Ghuj4
